app/src/main/java/com/yterletskyi/happyfriend/MainActivity.kt

```diff
@@ -59,12 +59,13 @@ class MainActivity : AppCompatActivity() {
 
     private fun inflateGraph() {
         val graph = navController.navInflater.inflate(R.navigation.mobile_navigation)
-
+        val args = Bundle()
+        args.putInt("title", R.string.pin_enter_title)
         val startDestination = pinCodeController.getPinCode()
             ?.let { R.id.pinScreen }
             ?: R.id.setupPinScreen
 
         graph.setStartDestination(startDestination)
-        navController.setGraph(graph, intent.extras)
+        navController.setGraph(graph, args)
```

Comment:
can we please add the "title" parameter only for `R.id.pinScreen` screen. We do not need it for `R.id.setupPinScreen` right ?